include/pybind11/cast.h

```diff
@@ -1870,7 +1870,12 @@ struct arg_v : arg {
 #if !defined(NDEBUG)
         , type(type_id<T>())
 #endif
-    { }
+    {
+      // Workaround! See https://github.com/pybind/pybind11/issues/2336
+      if (PyErr_Occurred()) {
+        PyErr_Clear();
```

Comment:
4-indent?

Reply:
Well spotted! Funny CI didn't catch that?

Reply:
Done.

Reply:
We don't run clang-format.